app/src/components/learn/learn-card.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+  CardFooter,
+} from "~/components/ui/card";
+
+import LinkExternal from "~/components/icons/link-external-02.1.svg";
+
+import { LearningResource } from "~/lib/validation/learn";
+
+export function LearnCard({ resource }: { resource: LearningResource }) {
+  return (
+    <Card
+      key={resource.title}
+      className="flex w-full shadow-md md:w-[calc((100%-.5rem)*.99)] lg:w-[calc((100%-1rem)*.45)] 2xl:w-[calc((100%-1.5rem)*.33)]"
```

Comment:
Here is how you can change the styles so that you don't get the weird width breaks and too much space on the right hand side:
```html
className="flex w-full shadow-md md:w-[calc((100%-2rem)/2)] xl:w-[calc((100%-4rem)/3)] 2xl:w-[calc((100%-6rem)/4)]"
```

To better understand this, here are the changes and their affects:

1. `md:w-[calc((100%-2rem)/2)]` - this utilizes the width of the gap and the number of items we want inside the row to determine the individual item width for medium sized screens
2. `xl:w-[calc((100%-4rem)/3)]` - because for a xl screen we want to have 3 items, we subtract twice the gap width (there will be two gaps between 3 items) and divide by 3 for calculating an individual items width
3. `2xl:w-[calc((100%-6rem)/4)]` - for ultrawide and 4k screens we will have 4 items in a row so we use the same logic as above